lua/octo/constants.lua

```diff
@@ -3,6 +3,7 @@ local api = vim.api
 local M = {}
 
 M.OCTO_COMMENT_NS = api.nvim_create_namespace("octo_marks")
+M.OCTO_HEADER_NS = api.nvim_create_namespace("octo_header_marks")
```

Comment:
Do we need an extmark for the header?

Reply:
Closing in regards of [this comment](https://github.com/pwntester/octo.nvim/pull/116#issuecomment-789578727).

Reply:
Closing in regards of [this comment](https://github.com/pwntester/octo.nvim/pull/116#issuecomment-789578727).